src/main/java/games/strategy/triplea/util/BonusIncomeUtils.java

```diff
@@ -8,26 +8,26 @@
 import games.strategy.util.IntegerMap;
 
 
-public class AiBonusIncomeUtils {
+public class BonusIncomeUtils {
 
   /**
    * Add AI bonus income based on the player's set percentage for all resources.
```

Comment:
Was thinking this may belong in an AI specific package, but then again, making this AI specific perhaps is needless. If not AI specific, then the javadoc here perhaps could use updating to remove the "AI" mention.